operations/companies.md

```diff
@@ -353,16 +231,16 @@ Options of the company.
 
 | Property | Type | Contract | Description |
 | :-- | :-- | :-- | :-- |
-| `Invoiceable` | boolean | required |  |
-| `AddFeesToInvoices` | boolean | required |  |
-| `AddTaxDeductedPaymentToInvoices` | boolean | required |  |
+| `Invoiceable` | boolean | required | Whether the company is invoiceable or not. |
+| `AddFeesToInvoices` | boolean | required | Whether the company has an additional fee applied for invoicing or not. |
+| `AddTaxDeductedPaymentToInvoices` | boolean | required | Whether tax-deducted payments should be automatically added to invoices. |
 
 #### Credit Rating Parameters
 Credit rating to define creditworthiness of the company.
 
 | Property | Type | Contract | Description |
 | :-- | :-- | :-- | :-- |
-| `Basic` | string | optional |  |
+| `Basic` | string | optional | Indicates the credit status of a company, determining if they can book services (CreditOk), must pay upfront (PaymentRequiredUpfront), or require local approval (LocalDecisionRequired). |
```

Comment:
issue: This should have been annotated with `EnumDataType` 🤔

Reply:
[fixed here](https://github.com/MewsSystems/gitbook-connector-api/pull/623/commits/b9ac60d6255e0060717460487df67ddc9e9b058a)